llmfoundry/callbacks/hf_checkpointer.py

```diff
@@ -76,6 +76,11 @@ def _maybe_get_license_filename(
 
     If the license file does not exist, returns None.
     """
+    # Early return if no local directory exists
```

Comment:
this should never happen right?

Reply:
assuming that is correct, please remove

Reply:
I hit this error before when the `local_pretrained_dir` was empty, so I added this early return. This seems like syntactically expected behavior though -- why not keep this check?

Reply:
I think my point is just that this function doesn't expect to be called if the dir is empty, so I'd be worried about it hiding an error upstream by returning silently.

Reply:
medium conviction opinion, would remove the check (or error if the dir is empty)

Reply:
cool i'll error instead